access-control-app/project.clj

```diff
@@ -59,4 +73,12 @@
                                         :let [project-dir (.replace (name group-artifact) "cmr-" "")]]
                                     ["shell" "ln" "-s" (str "../../" project-dir) "checkouts/,"]))
             ;; Alias to test2junit for consistency with lein-test-out.
-            "test-out" ["test2junit"]})
+            "test-out" ["test2junit"]
+            ;; Linting aliases
+            "kibit" ["do" ["with-profile" "lint" "shell" "echo" "== Kibit =="]
+                          ["with-profile" "lint" "kibit"]]
+            "eastwood" ["with-profile" "lint" "eastwood" "{:namespaces [:source-paths]}"]
+            "bikeshed" ["with-profile" "lint" "bikeshed"]
```

Comment:
Doesn't just adding the plugin automatically cause lein bikeshed to run with the bikeshed profile? I think it is fine to leave it here explicitly, just wanted to see if there's something I'm missing.